server/sonar-db-dao/src/main/java/org/sonar/db/purge/PurgeCommands.java

```diff
@@ -56,13 +54,13 @@
     return purgeMapper.selectAnalysisIdsAndUuids(query);
   }
 
-  void deleteAnalyses(String rootUuid) {
+  void deleteAnalyses(String rootComponentUuid) {
     profiler.start("deleteAnalyses (events)");
-    purgeMapper.deleteEventsByComponentUuid(rootUuid);
+    purgeMapper.deleteEventsByComponentUuid(rootComponentUuid);
     session.commit();
     profiler.stop();
 
-    List<List<String>> analysisUuidsPartitions = Lists.partition(IdUuidPairs.uuids(purgeMapper.selectAnalysisIdsAndUuids(new PurgeSnapshotQuery().setComponentUuid(rootUuid))),
+    List<List<String>> analysisUuidsPartitions = Lists.partition(IdUuidPairs.uuids(purgeMapper.selectAnalysisIdsAndUuids(new PurgeSnapshotQuery().setComponentUuid(rootComponentUuid))),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 184 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)